src/Organizations/OrganizationForm/OrganizationForm.js

```diff
@@ -87,7 +102,7 @@ const OrganizationForm = ({
     },
     {
       name: 'save',
-      handler: handleKeyCommand(handleSubmit, { disabled: pristine || submitting }),
+      handler: handleKeyCommand(handleSubmit, { disabled: pristine || submitting || isSubmitDisabled }),
```

Comment:
`pristine || submitting || isSubmitDisabled` is used in several places and keeps the same goal. Please create a const for it and reuse it in other places.